stdlib/public/runtime/Errors.cpp

```diff
@@ -231,6 +235,9 @@ reportNow(uint32_t flags, const char *message)
 #ifdef __APPLE__
   asl_log(nullptr, nullptr, ASL_LEVEL_ERR, "%s", message);
 #endif
+#ifdef __ANDROID__
+  __android_log_print(ANDROID_LOG_ERROR, "Swift", "%s", message);
```

Comment:
A few quick things:

- This should be `ANDROID_LOG_FATAL`
- I'm not sure how this tag is reported, but it seems like it should include slightly more info here from the uses I've found elsewhere
- Considering that these should be mutually-exclusive, I think a better structure would be `#if defined(...)/#elif defined(...)`